docs/guides/auth.md

```diff
@@ -326,3 +343,31 @@ $opts = [
 ];
 $client = new helloworld\GreeterClient('greeter.googleapis.com', $opts);
 ```
+
+###Authenticating with Google (Python)
+
+####Base case - No encryption/authentication
+```python
+channel = implementations.insecure_channel('localhost', 50051)
+stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+...
+```
+
+###Authenticate using OAuth2 token (legacy approach)
```

Comment:
:(. @anthmgoogle is there a timeline for this support? 

@soltanmm can you please make the fixes above and I'll LGTM (and reassign @LisaFC). Thanks!